config.go

```diff
@@ -700,6 +700,17 @@ type Config struct {
 	//
 	// Required: defaults to false
 	SkipTargetVerification bool
+	//
+	// The Ferry will enforce that the source DB has no Foreign Key constraints.
+	// It will error out before running.
+	//
+	// It can be overriden to skip this enforcement.
+	//
+	// This allows for custom implementations to circumvent
+	// issues that come with FK constraints.
+	//
+	// Required: defaults to false
+	SkipForeignKeyConstraintsCheck bool
```

Comment:
```suggestion
	// SkipForeignKeyConstraintsCheck is used to enforce that the 
	// source database has no Foreign Key constraints at runtime.
	// 
	// Setting the value to true will allow foreign keys on the source 
	// database during the run.
	//
	// Required: defaults to false
	SkipForeignKeyConstraintsCheck bool
```

Reply:
I like these changes, though there is one point I'd like to emphasize. I don't want to give the impression that by enabling this check, the ferry won't error out during runtime. 

I feel as though the statement: "Setting the value to true will allow foreign keys on the source database during the run", may suggest that Ghostferry _can_ handle foreign key constraints, which is false.

Reply:
I think I found a good description that makes the usage clearer:

Addressed in 8fe9a75